### PR TITLE
Update integrate.mdx

### PR DESCRIPTION
**Pull request by AnilMaktala**
_Issue [#4904](https://github.com/aws-amplify/docs/issues/4904)

_Description of changes:_
fix (android): corrected Update a Todo query sample code

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/fragments/start/getting-started/android/integrate.mdx [hunk 1]
```diff
@@ -476,34 +476,34 @@ You may want to change the contents of a record. Below, we'll query for a record
...
+  Amplify.DataStore.query(Todo::class.java, Where.matches(Todo.NAME.eq("Finish quarterly taxes")),
       { matches ->
           if (matches.hasNext()) {
               val todo = matches.next()
-              val updatedTodo = original.copyOfBuilder()
-                  .title("File quarterly taxes")
+              val updatedTodo = todo.copyOfBuilder()
+                  .name("File quarterly taxes")
                   .build()
               Amplify.DataStore.save(edited,
```
Comment (eeatonaws) [inline src/fragments/start/getting-started/android/integrate.mdx:508]:
```suggestion
              Amplify.DataStore.save(updatedTodo,
```

File: src/fragments/start/getting-started/android/integrate.mdx [hunk 2]
```diff
@@ -476,34 +476,34 @@ You may want to change the contents of a record. Below, we'll query for a record
...
-                  .build();
-              Amplify.DataStore.save(updatedTodo,
-                  updated -> Log.i("Tutorial", "Updated item: " + updatedTodo.getName(),
-                  failure -> Log.e("Tutorial", "Update failed.", failure)
-              );
-          }
-      },
-      failure -> Log.e("Tutorial", "Query failed.", failure)
-  );
+  Amplify.DataStore.query(Todo.class, Where.matches(Todo.NAME.eq("Finish quarterly taxes")),
```
Comment (eeatonaws) [inline src/fragments/start/getting-started/android/integrate.mdx:479]: nit: The indentation of this code block looks off. I think the original indentation looked okay.

**Review by eeatonaws — CHANGES_REQUESTED** [review]
The updated code LGTM! Left a comment on one other code change. Also, in the "Delete a Todo" section on this page, there's an error in the Java code (on line 542, `todo` should be `toDeleteTodo`). Could you make that change as part of this PR?

File: src/fragments/start/getting-started/android/integrate.mdx (hunk 2 above)
Comment (AnilMaktala) [reply src/fragments/start/getting-started/android/integrate.mdx]: @eeatonaws corrected the indentation.

Comment (AnilMaktala) [reply src/fragments/start/getting-started/android/integrate.mdx]: updated the delete code snippet.

**Review by eeatonaws — CHANGES_REQUESTED** [review]
On line 508, in `Amplify.DataStore.save`, `edited` should be replaced with `updatedTodo`.

**Review by hdworld11 — DISMISSED** [review]

File: src/fragments/start/getting-started/android/integrate.mdx (hunk 1 above)
Comment (AnilMaktala) [reply src/fragments/start/getting-started/android/integrate.mdx]: updated.

**Review by hdworld11 — APPROVED** [review]

**Review by katiegoines — APPROVED** [review]

**Pull request merged**